src/renderer/store/SsoAuthProvider.js

```diff
@@ -142,8 +46,19 @@ export const useSsoAuth = function () {
        */
       startAuthorization(options) {
         if (!pr.loading) {
+          pr.reset(true);
+
           // must not be loading (we can only handle one load at a time)
-          _startAuthorization(options);
+          try {
+            ssoUtil.startAuthorization(options);
+          } catch (error) {
+            pr.loading = false;
+            pr.loaded = true;
+            pr.error = error;
```

Comment:
As I mentioned on the original PR, `error` is an object here, while `pr.error` is a string, and it's user-facing, so this should be:
```suggestion
pr.error = strings.ssoAuthProvider.error.basicOnly();
```